app/web/features/auth/locales/en.json

```
@@ -182,6 +182,23 @@
     "error_message": "Error unsubscribing: {{message}}",
     "missing_payload_or_sig": "Invalid URL, missing payload or signature"
   },
+  "change_phone": {
+    "title": "Phone number",
+    "no_phone_description": "You have not added a phone number to your account. To add one, enter it below. We will verify it with a text message.",
+    "phone_label": "Your phone number",
+    "add_button_text": "Add phone number",
+    "add_success": "Phone added.",
+    "phone_not_verified_description": "We sent you a code to <1>{{phone}}</1>. To verify your number, please enter the code below:",
+    "code_label": "Verification code",
+    "wrong_code": "The code is wrong.",
+    "verify_button_text": "Verify phone number",
+    "verify_success": "Phone verified.",
+    "remove_phone_description": "Your phone number is currently <1>{{phone}}</1>. You can remove your phone number below, you will loose verification.",
```

```suggestion
    "remove_phone_description": "Your phone number is currently <2>{{phone}}</2>. You can remove your phone number below, you will loose verification.",
```
Think it needs to be this for the phone number to show up correctly